### ISSUE TITLE
Whorled appearance on breast biopsy

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10126]
**Preferred term label:**
Whorled appearance on breast biopsy

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Phyllodes

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001050
- label: Whorled
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: 
- def: Applies to an abnormality whose distribution and appearance resembles series of a concentric circles or spirals.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10126
- synonyms:
